controller/server/info.ml

```diff
@@ -46,9 +46,26 @@ let of_file f =
   |> Mustache.of_string
   |> return
 
+(** Format a 32-character string as groups of 4; leave other strings unchanged. *)
+let format_machine_id s =
+  if String.length s == 32 then
```

Comment:
According to `man machine-id`, a `machine-id` is exactly 32 characters long:

> The /etc/machine-id file contains the unique machine ID of the local system that is set during installation or boot. The machine ID is a single newline-terminated, hexadecimal, 32-character, lowercase ID. When decoded from hexadecimal, this corresponds to a 16-byte/128-bit value. This ID may not be all zeros.

Reply:
So, I would be in favor to remove this condition.

Reply:
I recently dealt with a string formatting function deep in a project that assumed a certain shape and applied a specific transformation without verifying its assumptions, producing very garbled serial numbers when their shape changed.

So I think checking preconditions is a good thing, even if it is superfluous according to our current expectations.